README.md

```diff
@@ -1,37 +1,78 @@
 # Nuxeo ONLYOFFICE integration plugin
 
-This plugin enables users to edit office documents from [Nuxeo](https://www.nuxeo.com/) using ONLYOFFICE Document Server.
+This plugin enables users to edit office documents from [Nuxeo](https://www.nuxeo.com/) using ONLYOFFICE Document Server - [Community or Integration Edition](#onlyoffice-document-server-editions).
 
 ## Features
 
+The plugin allows to:
 
+* Create and edit text documents, spreadsheets, and presentations.
+* Share documents with other users.
+* Co-edit documents in real-time: use two co-editing modes (Fast and Strict), Track Changes, comments, and built-in chat.
 
-## Installing ONLYOFFICE Document Server
+Supported formats:
 
-You will need an instance of ONLYOFFICE Document Server that is resolvable and connectable both from Nuxeo and any end clients. If that is not the case, use the official ONLYOFFICE Document Server documetnation page: [Document Server for Linux](http://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx). ONLYOFFICE Document Server must also be able to POST to Nuxeo directly.
+* For viewing and editing: DOCX, XLSX, PPTX.
+* For viewing only: PDF, ODT, ODS, ODP, DOC, XLS, PPT.
 
-The easiest way to start an instance of ONLYOFFICE Document Server is to use [Docker](https://github.com/onlyoffice/Docker-DocumentServer).
+## Installing ONLYOFFICE Document Server
 
+You will need an instance of ONLYOFFICE Document Server that is resolvable and connectable both from Nuxeo and any end-clients. ONLYOFFICE Document Server must also be able to POST to Nuxeo directly.
 
-## Compiling Nuxeo ONLYOFFICE integration plugin
+You can install free Community version of ONLYOFFICE Document Server or scalable enterprise-level Integration Edition.
 
+To install free Community version, use [Docker](https://github.com/onlyoffice/Docker-DocumentServer) (recommended) or follow [these instructions](https://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx) for Debian, Ubuntu, or derivatives.  
 
+To install Integration Edition, follow instructions [here](https://helpcenter.onlyoffice.com/server/integration-edition/index.aspx).
 
+Community Edition vs Integration Edition comparison can be found [here](#onlyoffice-document-server-editions).
 
 ## Installing Nuxeo ONLYOFFICE integration plugin
 
+Install it from [marketplace](https://connect.nuxeo.com/nuxeo/site/marketplace).
 
+You can also install it using [nuxeoctl](https://doc.nuxeo.com/nxdoc/installing-a-new-package-on-your-instance/).
+```
+nuxeoctl mp-install /path/to/onlyoffice-nuxeo-package-x.x.zip
+```
 
-## Configuring Nuxeo CONLYOFFICE integration plugin
+## Configuring Nuxeo ONLYOFFICE integration plugin
 
+Edit [nuxeo.conf](https://doc.nuxeo.com/nxdoc/configuration-parameters-index-nuxeoconf/) and add following lines:
+```
+onlyoffice.docserv.url=http://documentserver/
+onlyoffice.jwt.secret=
```

Comment:
add links what is jwt and how to enable in DocumentServer

Reply:
requested info has been added